src/tof_camera_node.cpp

```diff
@@ -102,7 +102,8 @@ class TofNode : public rclcpp::Node {
     void timer_callback() {
         if (m_streamOnFlag) {
             getNewFrame(camera, frame);
-            publishers.updatePublishers(camera, frame);
+	        rclcpp::Time TimeStamp= this->get_clock()->now();
```

Comment:
nit: extra space around `=`

Reply:
add additional space around = it is addressed in the most recent commit